_resources/port1.0/group/php-1.1.tcl

```diff
@@ -70,8 +70,13 @@ proc php._set_branches {option action args} {
                 test {}
                 destroot {
                     xinstall -d -m 0755 ${destroot}${prefix}/share/doc/${subport}
-                    system "echo \"${subport} is a stub port\" > ${destroot}${prefix}/share/doc/${subport}/README"
+                    system "echo $name is a stub port > ${destroot}${prefix}/share/doc/${name}/README"
```

Comment:
There's no need for any of the changes on this line. It works as is.

Reply:
Sorry. I will revert that.